build.sbt

```diff
@@ -49,11 +49,13 @@ libraryDependencies ++= Seq(
   "org.hsqldb" % "hsqldb" % "2.6.1",
   "com.sendgrid" % "sendgrid-java" % "2.2.2",
   "ch.qos.logback" % "logback-classic" % "1.2.10",
-  //---------- Test libraries -------------------//
+  "org.broadinstitute.dsde.workbench" %% "sam-client" % "0.1-e57a8f1-SNAP",
```

Comment:
Reminder to change this once https://github.com/broadinstitute/sam/pull/1148 is merged into develop and use the non-snapshot version of the client library